### ISSUE TITLE
Failed to start recording, Input device not found [iOS]

### ISSUE THREAD

**Issue by okanulkr** [llfbandit/record#233]
**Package version**
^5.0.1

**Describe the bug**
```
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: PlatformException(record, Failed to start recording, Input device not found., null)
#0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:651:7)
#1      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:334:18)
<asynchronous suspension>
#2      AudioRecorder.start (package:record/src/record.dart:60:5)
<asynchronous suspension>
```

**To Reproduce**
```
final record = AudioRecorder();
await record.start(const RecordConfig(), path: 'aFullPath/myFile.m4a');
```

**Smartphone (please complete the following information):**
 - Device: iPhone15 Simulator
 - OS: iOS17.0
 
**Additional context**
Working perfectly before updating to `^5.0.1` from `^4.4.4`

**Comment by HamadaAllipy** [comment]
I have the same issue

**Comment by llfbandit** [comment]
Should be fixed in `record_darwin: ^1.0.1-beta.1`. You will need to add it manually to test it.

**Comment by llfbandit** [comment]
Closing as duplicate of #222
